tahrir/views.py

```diff
@@ -70,6 +63,15 @@ def _get_user(request, id_or_nickname):
             return None
 
 
+def _get_team(request, team_id):
+    '''Get team for the given team_id'''
+    team = request.db.get_team(team_id=team_id)
+
+    if team:
+        return team
+    return None
```

Comment:
Could the db return you things like ``''`` or ``False`` ? If not, then I think returning team directly is just as good :)